public/mobile-app/src/routes/+page.svelte

```diff
@@ -1,276 +1,93 @@
-  }
+  return Response.redirect(`${url}?${params}`)
 }
 </script>
 
 <div>
-{#if !isAppInstalled}
-  <Installation />
-{:else}
-	<div>
-		{#if userEmailState}
-			<h1>Bienvenue {userEmailState} sur l'application AMI</h1>
-		{:else}
-			<h1>Bienvenue sur l'application AMI</h1>
-		{/if}
+{#if !isFranceConnected}
+	<h1>Bienvenue sur l'application AMI</h1>
 
+	<div class="fr-connect-group">
 		<button
+				class="fr-connect"
 				type="button"
-				onclick={askForNotificationPermission}
-				disabled={isAuthenticatedForNotifications}
+				id="fr-connect-button"
+				onclick={franceConnect}
 		>
-			S'authentifier pour recevoir des notifications
+			<span class="fr-connect__login">S’identifier avec</span>
+			<span class="fr-connect__brand">FranceConnect</span>
 		</button>
-		<span id="authentication-status" title="authentication-status-title">{authenticationStatus}</span>
-
-		<div>
-			<p>
-				<label
-				>Email
-					<input
-							bind:value={registerEmailInputValue}
-							type="text"
-							id="register-email"
-							name="register-email"
-					/>
-				</label>
-			</p>
-			<p>
-				<span id="email-validation-status">{userEmailValidationStatus}</span>
-			</p>
-			<p>
-				<button
-						type="button"
-						id="register-with-ami"
-						onclick={registerWithAmi}
-						disabled={isRegisteredWithAmi}
-				>
-					S'enregistrer auprès d'AMI
-				</button>
-				<span id="registration-status">{registrationStatus}</span>
-			</p>
-		</div>
-
-		<div>
-			<h2>Historique des notifications</h2>
-			<button
-					type="button"
-					onclick={retrieveNotifications}
-			>
-				Rafraîchir la liste des notifications reçues
-			</button>
-			{#if userNotifications.length === 0}
-				<p>Vous n'avez pas reçu de notification pour l'instant</p>
-			{:else}
-				<ul>
-					{#each userNotifications as notification}
-						<li>
-							<p>Notification reçue le {notification.formattedDate}</p>
-							<p>Titre : {notification.title}</p>
-							<p>Message : {notification.message}</p>
-							<p>Expéditeur : {notification.sender}</p>
-						</li>
-					{/each}
-				</ul>
-			{/if}
-		</div>
+		<p>
+			<a href="https://franceconnect.gouv.fr/" target="_blank" rel="noopener" title="Qu’est-ce que FranceConnect ? - nouvelle fenêtre">Qu’est-ce que FranceConnect ?</a>
+		</p>
 	</div>
+{:else}
+  <h1>Bonjour { userinfo.given_name }</h1>
+
+  <ul>
+    <li>userinfo: { userinfo }</li>
+    <li>sub: { userinfo.sub }</li>
+    <li>given_name: { userinfo.given_name }</li>
+    <li>given_name_array: { userinfo.given_name_array }</li>
+    <li>family_name: { userinfo.family_name }</li>
+    <li>birthdate: { userinfo.birthdate }</li>
+    <li>gender: { userinfo.gender }</li>
+    <li>birthplace: { userinfo.birthplace }</li>
+    <li>birthcountry: { userinfo.birthcountry }</li>
+    <li>email: { userinfo.email }</li>
+    <li>aud: { userinfo.aud }</li>
+    <li>exp: { userinfo.exp }</li>
+    <li>iat: { userinfo.iat }</li>
+    <li>iss: { userinfo.iss }</li>
+  </ul>
```

Comment:
Nice!
This could also be written something like the following to make sure that everything we got back from FC is printed out, so if there are any changes in the future, we'd still see them
```suggestion
  <preformat>
    {JSON.stringify(userinfo, null, 2)}
  </preformat>
```

I haven't tried it, so this pseudocode might need some changes, also I understand this is just debug information that won't stay in the "final" version, so feel free to just ignore this comment.

Reply:
So i've slightly modified your code so the first list item uses the `JSON.stringify` hack, as it was otherwise displaying `[object object]`, which wasn't very useful ;)

We now have the best of both, but we might want to reduce the amount of debug output in the future.